vivisect/tests/testvivisect.py

```diff
@@ -410,9 +410,9 @@ def test_vaset_populate(self):
                                  (134553488,), (134556128,), (134559504,), (134573472,), (134554096,), (134572464,),
                                  (134575616,), (134575872,), (134575552,), (134582112,), (134561824,), (134573264,),
                                  (134561232,), (134574816,), (134547936,), (134582096,), (134552448,), (134552560,),
-                                 (134575584,), (134553216,), (134573760,), (134582064,), (134583824,), (134519952,),
```

Comment:
and these?

Reply:
All of them are the import shims around elf imports (since these are the emucode ones)

So some are like lxstat and other functions that are basically just shims around things like bindtextdomain. Nothing terrible to have not be an emucode function.

Reply:
so we get these some other way (from ELF parser)?  or we don't have them identified at all?

Reply:
We end up getting them via codeflow or some other mechanism. They're definitely there. With the fix I just threw up, the opcode count and everything else didn't change. We're just discovering them earlier in the analysis passes now.